artifactory_test.go

```diff
@@ -874,15 +880,18 @@ func TestArtifactorySetPropertiesExcludeByCli(t *testing.T) {
 func TestArtifactoryDeleteProperties(t *testing.T) {
 	initArtifactoryTest(t)
 	artifactoryCli.Exec("upload", "testsdata/a/a*.in", tests.Repo1+"/")
-	artifactoryCli.Exec("sp", tests.Repo1+"/*", "prop=val")
+	artifactoryCli.Exec("sp", tests.Repo1+"/*", "prop=val;key=value")
 	artifactoryCli.Exec("delp", tests.Repo1+"/*", "prop")
 	resultItems := searchItemsInArtifacotry(t)
+	if len(resultItems) == 0 {
+		t.Error("No artifacts were found.")
+	}
 
 	for _, item := range resultItems {
 		properties := item.Properties
 		for _, prop := range properties {
-			if prop.Key != "" {
-				t.Error("Wrong properties")
+			if prop.Key == "prop" {
+				t.Error("Property prop were not deleted from artifact", item.Name)
```

Comment:
Property prop were not deleted from artifact --> Property 'prop' was not deleted from artifact

Reply:
Done